plugins/emoji/plugin.js

```diff
@@ -67,7 +67,8 @@
 
 				function matchCallback( text, offset ) {
 					var left = text.slice( 0, offset ),
-						match = left.match( new RegExp( ':\\S{' + charactersToStart + '}\\S*$' ) );
+						// Positive lookbehind for space, as emoji should be started with space or newline (#2195).
+						match = left.match( new RegExp( '(?<=\\s\|^):\\S{' + charactersToStart + '}\\S*$' ) );
```

Comment:
Unfortunately lookbehind and lookahead RegExp are not widely supported, because of that tests are failing on IE11, Firefox, Safari.
You can try to use groups and/or non-capturing groups, but still it will require changing way how start value is set up.